ml-agents-envs/mlagents_envs/communicator_objects/unity_to_external_pb2_grpc.py

```diff
@@ -4,7 +4,7 @@
 from mlagents_envs.communicator_objects import unity_message_pb2 as mlagents__envs_dot_communicator__objects_dot_unity__message__pb2
 
 
-class UnityToExternalProtoStub:
+class UnityToExternalProtoStub(object):
```

Comment:
Follow up jira: https://jira.unity3d.com/browse/MLA-1215 
We should have run the proto check test when these were modified.